### ISSUE TITLE
Community Poll: Sorcery Logo

### ISSUE THREAD

**Issue by athix** [Sorcery/sorcery#259]
Hello world! As a part of Sorcery v1 I'm looking to revitalize the documentation, as well as the general meta-information around Sorcery. One part of this has been creating a logo for Sorcery. I'm hoping that having an actual logo will help Sorcery feel more like a fully fledged project, as well as indicate that it will continue to be supported for the years to come. Along that vein, I was hoping to get the community's opinion on some of the different options available. Anyone and everyone is welcome to give their opinion on these, so please comment with your thoughts!

---
![Socery Logo Final Drafts_Page_1](https://user-images.githubusercontent.com/1802598/96352101-4e5df100-10b0-11eb-94c8-81fb8e4004ec.png)

---

![Socery Logo Final Drafts_Page_2](https://user-images.githubusercontent.com/1802598/96352102-4f8f1e00-10b0-11eb-9088-7ba3ba9f12b2.png)

---

![Socery Logo Final Drafts_Page_3](https://user-images.githubusercontent.com/1802598/96352103-4f8f1e00-10b0-11eb-8b91-3c27e0545eb5.png)

---

![Socery Logo Final Drafts_Page_4](https://user-images.githubusercontent.com/1802598/96352104-5027b480-10b0-11eb-8b63-ffceb7bae599.png)

---

![Socery Logo Final Drafts_Page_5](https://user-images.githubusercontent.com/1802598/96352105-50c04b00-10b0-11eb-9a23-b45a96885a65.png)

---

![Socery Logo Final Drafts_Page_6](https://user-images.githubusercontent.com/1802598/96352106-50c04b00-10b0-11eb-9acf-7e6704c323fa.png)

**Comment by wout** [comment]
@athix I really love you're grabbing the bull by the horns. :)

Option 1.2b appeals most to me. I like that it has limited colours and has a slight variation in the size of the "balls", which gives the illusion of depth. From all the examples, it appears most confident and strong.

A bit unrelated, but I was wondering if you have considered starting a community. Being able to quickly ask Sorcery-related questions is something I really miss. Opening a Github issue for simple questions seems overkill. Discord works well, but I know not everybody is as happy creating an account there.

Finally, I don't have a lot of time on my hands, but I'd like to lend a hand. So if there is something I can help you with, please let me know.

**Comment by athix** [comment]
> A bit unrelated, but I was wondering if you have considered starting a community. Being able to quickly ask Sorcery-related questions is something I really miss. Opening a Github issue for simple questions seems overkill. Discord works well, but I know not everybody is as happy creating an account there.

I think having a community chat makes sense, although I hesitate setting one up just quite yet. I want to focus on getting v1 and the documentation up to snuff first. We currently have the Gitter chat channel, but I'll be honest that I don't really keep up with it at all. Discord would be a great and accessible alternative, imo. Like you said, it's just a question of how many people would be off-put by having to create a Discord account.

> Finally, my time is limited but I'd like to lend a hand. So if there is something I can help you with, please let me know.

There isn't too much to do quite yet, because a lot of it is blocked by getting v1 shipped. My current plan is to get a side-project to a good stopping point, then switch over to focusing on v1 again. I've intentionally made getting Sorcery v1 shipped a blocker for that project, so it's just a question of how much time it takes complete. Perhaps in the meantime you could take a look at what I've got so far, and provide some feedback? ([sorcery/sorcery-rework](https://github.com/sorcery/sorcery-rework)) If you wanted to try tackling converting some of the other submodules to the new plugin format, that would also be a help in the interim till I can get back to focusing on primarily Sorcery.

**Comment by wout** [comment]
> I think having a community chat makes sense, although I hesitate setting one up just quite yet. I want to focus on getting v1 and the documentation up to snuff first. We currently have the Gitter chat channel, but I'll be honest that I don't really keep up with it at all.

Yeah, it's not a priority. Having v1 shipped is more important. The Gitter channel is as good as dead by the looks of it. I have the feeling Gitter is on its way out anyway since so many communities are moving elsewhere.

Are you doing a separate site for documentation or keeping it on Github?

> Perhaps in the meantime you could take a look at what I've got so far, and provide some feedback? (sorcery/sorcery-rework) If you wanted to try tackling converting some of the other submodules to the new plugin format, that would also be a help in the interim till I can get back to focusing on primarily Sorcery.

That looks like a good place to start. I'll look at it over the weekend.

**Comment by olimart** [comment]
Nice.
Would go with either 1.1a or 1.1b
1.1b is more subtle so it may be easier to apply in different contexts.
(can't find the difference between `1.1a` vs `1.2a` & `1.1b` vs `1.2b`)

**Comment by olimart** [comment]
> One thing I really like about well-maintained libraries is when their Getting Started documents really do have everything you need to get started.

Something I always appreciate with Sorcery which you don't find on all others get started is the API summary section broken down by module. Very useful.

**Comment by athix** [comment]
Thanks everyone for your feedback! I passed it by some others as well, and it seems the general consensus is 1.2b.
